app/models/user.rb

```diff
@@ -1,9 +1,27 @@
 class User < ApplicationRecord
+  has_secure_password
+
   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
   before_save {self.email = email.downcase}
```

Comment:
remove this file, you already define at line 10

Reply:
sorry but which line should I delete? is it has_secure_password?